### DISCUSSION TITLE
Editor text replacement

### DISCUSSION THREAD

**Discussion by Vuurvos1** [cssbattle/forum#80] [Bugs reports]
**Bug description**
When selecting a letter in the text editor and typing the same letter again it doesn't get replaced

**How to recreate**
1. Type a letter in the text editor
2. Select the single letter
3. Hit the same letter (multiple times) on your keyboard
4. Letter doesn't get replaced

**Expected behaviour**
The letter should get replaced so you can continue typing

**Note** 
that this is very minor but is something I noticed

**Comment by chinchang** [comment]
Ahh, this is bad. Seems like a bug in react-codemirror2. Adding bug link for reference - https://github.com/scniro/react-codemirror2/issues/255
